sda-commons-server-morphia-example/src/test/java/org/sdase/commons/server/morphia/example/MorphiaApplicationIT.java

```diff
@@ -59,7 +59,7 @@ public void before() {
   public void shouldStoreCarEntity() {
     addData();
     assertThat(datastore.createQuery(Car.class).count()).isEqualTo(2);
-    assertThat(datastore.createQuery(Car.class).find())
+    assertThat(datastore.createQuery(Car.class).find().toList())
```

Comment:
This is related to the assertJ changes, right?

Reply:
Yes. Same change like in DW2-branch.